### PR TITLE
DOC-8241 Docs for Prometheus metrics integration for Dedicated clusters

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-8241

(1) On Export Metrics to Dedicated Cluster page, added Prometheus filter and sections. (2) Added Cloud release note with link to Export Metrics to Dedicated Cluster page with Prometheus filter. (3) On Monitor CockroachDB with Prometheus page for self-hosted, added tip with link to Export Metrics to Dedicated Cluster page.

Rendered page:
https://deploy-preview-18252--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-metrics.html?filters=prometheus-metrics-export

### CODE REVIEW

**Review by dhartunian — APPROVED** [review]
LGTM

File: src/current/cockroachcloud/export-metrics.md [hunk 1]
```diff
@@ -214,11 +216,117 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
...
+    {% include_cached copy-clipboard.html %}
+    ~~~yaml
+    global:
+      scrape_interval: 10s
+      evaluation_interval: 10s
+
+    # Prometheus configuration for CockroachDB Dedicated for a single region
+    scrape_configs:
+      - job_name: "scrape-cockroach-us-east4"
+        metrics_path: '/api/v1/clusters/{cluster_id}/metricexport/prometheus/`{cluster_region}/scrape'
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics.md:309]: good catch! removed.

**Review by florence-crl — COMMENTED** [review]
TFTR

File: src/current/cockroachcloud/export-metrics.md [hunk 2]
```diff
@@ -214,11 +216,118 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
 
 </section>
 
+<section class="filter-content" markdown="1" data-scope="prometheus-metrics-export">
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID:
+
+	1. Visit the CockroachDB {{ site.data.products.cloud }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. It should resemble `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
```
Comment (rmloveland) [inline src/current/cockroachcloud/export-metrics.md:225]: Suggest changing slightly to be more explicit:

"The ID should resemble ..."

File: src/current/cockroachcloud/export-metrics.md [hunk 3]
```diff
@@ -214,11 +216,118 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
...
+
+1. Issue the following Cloud API command to enable metrics export for your CockroachDB {{ site.data.products.dedicated }} cluster:
```
Comment (rmloveland) [inline src/current/cockroachcloud/export-metrics.md:227]: "Cloud API" could be a link to https://www.cockroachlabs.com/docs/cockroachcloud/cloud-api

File: src/current/cockroachcloud/export-metrics.md [hunk 4]
```diff
@@ -214,11 +216,118 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
...
+      "cluster_id": "f78b7feb-b6cf-4396-9d7f-494982d7d81e",
+      "user_message": "This integration is active.",
+      "status": "ENABLED",
+      "targets": {
+        "us-east4": "https://cockroachlabs.cloud/api/v1/clusters/f78b7feb-b6cf-4396-9d7f-494982d7d81e/metricexport/prometheus/us-east4/scrape"
+      }
+    }
+    ~~~
+
+    There is a separate scrape endpoint per region if you have a multi-region cluster.
```
Comment (rmloveland) [inline src/current/cockroachcloud/export-metrics.md:262]: "multi-region" could be a link to https://www.cockroachlabs.com/docs/stable/multiregion-overview

**Review by rmloveland — APPROVED** [review]
LGTM! 

Separately: I don't know much about the cloud release notes portion or why it's still saying Jan 17.  Let's ping some folks in our team channel for help

File: src/current/cockroachcloud/export-metrics.md (hunk 2 above)
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics.md]: Updated

File: src/current/cockroachcloud/export-metrics.md (hunk 3 above)
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics.md]: Added link

File: src/current/cockroachcloud/export-metrics.md (hunk 4 above)
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics.md]: added link

**Review by florence-crl — COMMENTED** [review]
TFTR

**Pull request merged**